### PR TITLE
Add missing Jmol element colors

### PR DESCRIPTION
**Pull request by cjh1**
This list was generated from: http://jmol.sourceforge.net/jscolors/index.en.html#Atoms

### CODE REVIEW

File: 3Dmol/colors.js [hunk 1]
```diff
@@ -286,34 +290,190 @@ $3Dmol.elementColors.Jmol = {
...
+        'Ra': 0x007D00,
+        'RA': 0x007D00,
+        'Ac': 0x70ABFA,
+        'AC': 0x70ABFA,
+        'Th': 0x00BAFF,
+        'TH': 0x00BAFF,
+        'Pa': 0x00A1FF,
+        'PA': 0x00A1FF,
+        'U': 0x008FFF,
+        'U': 0x008FFF,
```
Comment (alesgenova) [inline 3Dmol/colors.js]: Duplicated entry

File: 3Dmol/colors.js [hunk 2]
```diff
@@ -286,34 +290,190 @@ $3Dmol.elementColors.Jmol = {
         'Si': 0xF0C8A0,
         'SI': 0xF0C8A0,
         'P': 0xFF8000,
+        'P': 0xFF8000,
```
Comment (alesgenova) [inline 3Dmol/colors.js]: Duplicated

**Comment by cjh1** [comment]
@alesgenova Thanks for catching those.

**Comment by dkoes** [comment]
Thanks!

**Pull request merged**